modules/publicize/publicize-jetpack.php

```diff
@@ -541,10 +542,7 @@ function options_page_facebook() {
 			?>
 
 			<?php if ( ! empty( $me['name'] ) ) : ?>
-				<p><?php printf(
-						esc_html__( 'Publicize to my %s:', 'jetpack' ),
-						'<strong>' . esc_html__( 'Facebook Wall', 'jetpack' ) . '</strong>'
-					); ?></p>
+				<p><?php esc_html_e( 'Publicize to my <strong>Facebook Wall</strong>:', 'jetpack' ); ?></p>
```

Comment:
Unfortunately we cannot do this with %s as there are languages that have to align the translation of `my` with the variable. For example in German: "Auf meine_r_ Facebook Wall veröffentlichen." vs. "Auf meine_m_ Tumblr Blog veröffentlichen."